docs/cli/kops_create_cluster.md

```diff
@@ -14,31 +14,30 @@ kops create cluster
-      --cloud string                    Cloud provider to use - gce, aws
-      --dns-zone string                 DNS hosted zone to use (defaults to longest matching zone)
-      --image string                    Image to use
-      --kubernetes-version string       Version of kubernetes to run (defaults to version in channel)
-      --master-security-groups string   Add precreated additional security groups to masters.
-      --master-size string              Set instance size for masters
-      --master-zones string             Zones in which to run masters (must be an odd number)
-      --model string                    Models to apply (separate multiple models with commas) (default "config,proto,cloudup")
-      --network-cidr string             Set to override the default network CIDR
-      --networking string               Networking mode to use.  kubenet (default), classic, external, cni, kopeio-vxlan, weave, calico. (default "kubenet")
-      --node-count int                  Set the number of nodes
-      --node-security-groups string     Add precreated additional security groups to nodes.
-      --node-size string                Set instance size for nodes
-      --out string                      Path to write any local output
-      --project string                  Project to use (must be set on GCE)
-      --ssh-public-key string           SSH public key to use (default "~/.ssh/id_rsa.pub")
-      --target string                   Target - direct, terraform (default "direct")
-  -t, --topology string                 Controls network topology for the cluster. public|private. Default is 'public'. (default "public")
-      --vpc string                      Set to use a shared VPC
-      --yes                             Specify --yes to immediately create the cluster
-      --zones string                    Zones in which to run the cluster
+      --admin-access stringSlice    Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP. (default [0.0.0.0/0])
+      --associate-public-ip         Specify --associate-public-ip=[true|false] to enable/disable association of public IP for master ASG and nodes. Default is 'true'.
+      --bastion                     Pass the --bastion flag to enable a bastion instance group. Only applies to private topology.
+      --channel string              Channel for default versions and configuration to use (default "stable")
+      --cloud string                Cloud provider to use - gce, aws
+      --dns string                  DNS hosted zone to use: public|private. Default is 'public'. (default "Public")
+      --dns-zone string             DNS hosted zone to use (defaults to longest matching zone)
+      --image string                Image to use
+      --kubernetes-version string   Version of kubernetes to run (defaults to version in channel)
```

Comment:
I think we need --master-count docs for https://github.com/kubernetes/kops/pull/1710